### PR TITLE
update glider tracks

### PR DESCRIPTION
**Pull request by birdage**
@DanielJMaher @FBRTMaka im going to test this sp hold fire for now, just want to get the PR up. Ill need @FBRTMaka to test on production,  

TODO
- [ ] add ability to preload glider tracks similar to the still camera images
- [ ] subsequent requests add to position from last known date to the current date time
- [ ] only telemetered glider streams will be be used to update the tracks
- [ ] for the telemetered glider track, subsequent requests should use a data request limit of 10 data points / hour

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher will only update those gliders that have a newer end time in the `metadata/times` field when checked

**Comment by DanielJMaher** [comment]
@birdage This went stale but should be easy to bring up to speed.. this is an important one to have right though so @oceanzus should give it a good review.  I can discuss the goal of this PR with you when you are free Jim.

**Comment by birdage** [comment]
@DanielJMaher fixed the rebase untested.

File: ooiservices/app/uframe/controller.py [hunk 1]
```diff
@@ -376,64 +375,124 @@ def _compile_glider_tracks():
...
+
+    lat_field   = "latitude"
+    lon_field   = "longitude"
+    depth_field = 'int_ctd_pressure'
+    bar_to_m    = 0.09804139432
+
     #params for position and depth info
-    params = "?parameters=PD1335,PD1336,PD1276&limit=1000"
+    #only request "depth" as nominal lat,lon are passed anyway
+    params = "?parameters=PD7&limit=10"
```
Comment (DanielJMaher) [inline ooiservices/app/uframe/controller.py]: Interpolation is broken.  We need t revert to PD 1335 and 1336.  Also it doesnt look like you request depth anymore...

Side note, we need the limit to be over 10.  1000 should be good.

Comment (DanielJMaher) [inline ooiservices/app/uframe/controller.py]: We also need a backlog of high fidelity glider tracks for gliders that are not currently in the water

Comment (birdage) [inline ooiservices/app/uframe/controller.py]: we need a method then to 1)preload previous glider tracks then 2)subsequent requests use a lower limit/spatial fidelity

**Pull request closed without merging**